module/mailnews/newsread.c

```diff
@@ -118,7 +141,12 @@ INT_PTR CALLBACK ReadNewsDialogProc(HWND hDlg, UINT message, WPARAM wParam, LPAR
    NM_LISTVIEW *nm;
    
    char date[MAXDATE], title[MAX_SUBJECT + MAXDATE + MAXNAME + 10];
-   
+
+    // Load sort arrow bitmaps
+   hbmUpArrow = (HBITMAP)LoadImage(GetModuleHandle(NULL), MAKEINTRESOURCE(IDB_UPARROW), IMAGE_BITMAP, 0, 0, LR_LOADTRANSPARENT | LR_LOADMAP3DCOLORS);
+
+   hbmDownArrow = (HBITMAP)LoadImage(GetModuleHandle(NULL), MAKEINTRESOURCE(IDB_DOWNARROW), IMAGE_BITMAP, 0, 0, LR_LOADTRANSPARENT | LR_LOADMAP3DCOLORS);
```

Comment:
lines too long

Reply:
What's you preference here? How about this?

```
// Load sort arrow bitmaps
hbmUpArrow = (HBITMAP)LoadImage(GetModuleHandle(NULL), 
    MAKEINTRESOURCE(IDB_UPARROW), IMAGE_BITMAP, 0, 0, 
    LR_LOADTRANSPARENT | LR_LOADMAP3DCOLORS);

hbmDownArrow = (HBITMAP)LoadImage(GetModuleHandle(NULL), 
    MAKEINTRESOURCE(IDB_DOWNARROW), IMAGE_BITMAP, 0, 0, 
    LR_LOADTRANSPARENT | LR_LOADMAP3DCOLORS);
```

Reply:
Sure, that's fine.  Long lines are hard to read when they wrap in github.